src/main/java/org/broadinstitute/hellbender/tools/pon/coverage/pca/HDF5PCACoveragePoN.java

```diff
@@ -186,7 +187,8 @@ public double getVersion() {
     public double[] getTargetFactors() {
         final double[] values = file.readDoubleArray(TARGET_FACTORS_PATH);
         if (values.length != targetNames.get().size()) {
-            throw new GATKException(String.format("Wrong number of elements in the target factors recovered from file '%s': %d != %d", file.getFile(), values.length, targetNames.get().size()));
+            throw new GATKException(String.format("Wrong number of elements in the target factors recovered " +
+                    "from file '%s': %d != %d", file.getFile(), values.length, targetNames.get().size()));
```

Comment:
It would be more informative to replace "%d != %d" in the exception message with "(from file) %d != (number of targets) %d"